Scripts/SDL/EderNaybree.sdl

```diff
@@ -0,0 +1,27 @@
+STATEDESC EderNaybree
+{
+    VERSION 1
+    VAR AGETIMEOFDAY            AthsDayNight01[1]
+    VAR BOOL            VCatBookVis[1]        DEFAULT=0    DEFAULTOPTION=VAULT
+    VAR BOOL            ExtcolBookVis[1]        DEFAULT=0    DEFAULTOPTION=VAULT
+    VAR BOOL            RewardKelsFleeceVis[1]        DEFAULT=0    DEFAULTOPTION=VAULT
```

Comment:
We would appreciate all of the fields in the SDL file to be vertically aligned, like so:

```suggestion
    VAR AGETIMEOFDAY            AthsDayNight01[1]
    VAR BOOL                    VCatBookVis[1]            DEFAULT=0    DEFAULTOPTION=VAULT
    VAR BOOL                    ExtcolBookVis[1]          DEFAULT=0    DEFAULTOPTION=VAULT
    VAR BOOL                    RewardKelsFleeceVis[1]    DEFAULT=0    DEFAULTOPTION=VAULT
```